### ISSUE TITLE
Feature Request: Traefik + Caddy2

### ISSUE THREAD

**Issue by curator888** [SelfhostedPro/selfhosted_templates#89]
https://hub.docker.com/_/traefik

https://hub.docker.com/r/hectormolinero/caddy2

**Comment by RobLoach** [comment]
How does this container differ from the [officially supported one](https://hub.docker.com/_/caddy)? Looks like the official one has a lot more uses.

**Comment by Qoyyuum** [comment]
I agree. The official caddy is most likely better supported too. I'm not familiar with how to use Caddy, so no comment.

In regards to quick capability of traefik support is to define the routers as mentioned in their [forum thread](https://forums.portainer.io/t/solved-templates-when-using-traefik/464/5). Although traefik needs to be in and as part of the stack with portainer before using it I think. So when just `docker-compose`, the new containers needs to be on the same network as traefik and portainer as another dependency.

**Comment by curator888** [comment]
You're right @RobLoach its probably better, I was wanting to make sure though that it would be Caddy2 not Caddy as they recently released Caddy2 but are still currently maintaining both.
